src/csharp/examples/server/API/DoCoolStuff/CoolStuffModel.cs

```diff
@@ -0,0 +1,11 @@
+namespace server.Models
+{
+    public class CoolStuffModel
```

Comment:
Suggestion:
Use `record` rather than `class` when the class is meant to represent data only

A record improves the `Equals` checks meant for data rather than using the reference of the class

example:

```csharp
public record CoolStuffModel(string DoubleCool);
//or with custom accessors (get-only in this case)
public record CoolStuffModel {
    public string DoubleCool { get; } 
    public CoolStuffModel(string doubleCool) {
        DoubleCool = doubleCool;
    }
}
```

Extra: `record` in C# is like `data class` in kotlin, but with more flexibility in terms of inheritance

Reply:
nifty